buildSrc/overview-general.html

```diff
@@ -1,79 +1,11 @@
 <html lang="en">
 <body>
-<p style="text-align: center">
-	<a href="https://twitch4j.github.io">
-		<img alt="Twitch4J Logo" height="256" src="https://twitch4j.github.io/assets/images/logo.svg">
-	</a>
-</p>
-<h1 style="text-align: center">Welcome to the Twitch4J Javadoc Documentation</h1>
-<table style="border: 1px solid #000; margin-left: auto; margin-right: auto">
-	<caption style="padding-bottom: 6px">All packages have a specific groups contained into each module</caption>
-	<tr>
-		<th>Package name</th>
-		<th>Group Name</th>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-common</code></td>
-		<td>Common</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-auth</code></td>
-		<td>Auth</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-chat</code></td>
-		<td>Chat</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-eventsub-common</code></td>
-		<td>EventSub</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-graphql</code></td>
-		<td>GraphQL</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-pubsub</code></td>
-		<td>PubSub</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-extensions</code></td>
-		<td>Extensions API</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-helix</code></td>
-		<td>New API - Helix</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-kraken</code></td>
-		<td>Kraken API v5 <sup>(deprecated)</sup></td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j-messageinterface</code></td>
-		<td>Twitch Message Interface - API</td>
-	</tr>
-	<tr>
-		<td style="float: right"><code>twitch4j</code></td>
-		<td>Core</td>
-	</tr>
-</table>
-<h2>To getting started please refer to our documentation</h2>
+<p><img alt="Twitch4J Logo" width="64" height="64" src="https://twitch4j.github.io/assets/images/logo.svg"/></p>
+<p>Check out our documentation at <a href="https://twitch4j.github.io/">twitch4j.github.io</a></p>
 <ul>
-	<li><a href="#">Getting Started</a></li>
-	<li><a href="#">Basics Usage</a></li>
-	<li><a style="text-decoration: line-through" href="#">Android Compatibility</a> <sup>(in
-		progress)</sup></li>
-	<li><a href="#">Minecraft Plugin</a></li>
-</ul>
-<h2>Thank you to our sponsors to provide open-source developing tools</h2>
-<ul>
-	<li>
-		<a href="https://www.jetbrains.com/">JetBrains</a> - for providing <a
-			href="https://www.jetbrains.com/idea/">IntelliJ IDEA</a></li>
-	<li>
-		<a href="https://www.yourkit.com/">YourKit</a> - for providing <a
-			href="https://www.yourkit.com/java/profiler/">Java Profiler</a> for Performance Tuning
-	</li>
+	<li><a href="https://twitch4j.github.io/javadoc">Aggregated Javadocs (all modules)</a></li>
```

Comment:
It should be in `overview-single.html`. If someone gonna checks specific module in Javadoc.io it needs redirect to the main one.